### PR TITLE
#20-fix-broken-image

### PR DESCRIPTION
**Pull request by clarmso**
Add default profile image for a user that doesn't have an image. In addition, the default image is displayed when waiting for the profile image to be fetched.

### CODE REVIEW

**Review by ANDREYDEN — CHANGES_REQUESTED** [review]
Nice job! A small, yet functional change. There is currently an error being thrown regarding the `photoURL` being null in `profile.js:10`. It would be nice if that error is also addressed in this PR.

File: public/src/js/profile.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ async function populateProfile(user) {
 
     if (user) {
         document.getElementById('fullName').innerHTML = user.displayName;
-        document.getElementById('small_profile_image').setAttribute('src', userPhoto);
+        if (userPhoto)
+            document.getElementById('small_profile_image').setAttribute('src', userPhoto);
```
Comment (ANDREYDEN) [inline public/src/js/profile.js]: It is good practice to use `{ ... }` even with one-line `if` statements:
```suggestion
        if (userPhoto) {
            document.getElementById('small_profile_image').setAttribute('src', userPhoto);
        }
```

**Comment by clarmso** [comment]
Regarding `photoURL` being null: Do you mean `user`should be defined before calling `user.photoURL`? I've reorganized the code to guard against calling `user.photoURL`if `user` isn't defined. Please let me know if this is not what you meant.

**Review by ANDREYDEN — APPROVED** [review]
Looks good!

**Pull request merged**